.github/workflows/ci.yml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 The Error Prone Authors.
+# Copyright 2020-2024 The Google Java Format Authors
```

Comment:
This is a nit, but I think the guidance is to use the year the file was created, and that it doesn't have to be updated to the current year, so just '2020' should be sufficient.

Reply:
sure thing, done!